exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Table.kt

```diff
@@ -1214,6 +1214,21 @@ open class Table(name: String = "") : ColumnSet(), DdlAware {
         }
     }
 
+    @Deprecated(
+        message = "This will be removed in future releases when the check becomes default in IdentifierManagerApi",
+        level = DeprecationLevel.WARNING
+    )
+    private fun String.warnIfUnflaggedKeyword() {
+        val warn = TransactionManager.currentOrNull()?.db?.identifierManager?.isUnflaggedKeyword(this) == true
+        if (warn) {
+            exposedLogger.warn(
+                "Keyword identifier used: '$this'. Case sensitivity may not be kept when quoted by default: '${inProperCase()}'. " +
+                    "To keep case sensitivity, opt-in and set 'preserveKeywordCasing' to true in DatabaseConfig block."
+            )
```

Comment:
To reduce possibly overloading users' logs, a warning is now only logged during table creation for a keyword identifier iff the user has not opted-in for the flag.